### DISCUSSION TITLE
Make manual particles wobble around initial position?

### DISCUSSION THREAD

**Discussion by ldorigo** [matteobruni/tsparticles#5259] [Q&A]
I want to start with a list of initial particles that remain more or less centered around their initial position, plus some random particles that roam around freely. I tried setting `wobble` with `move` disabled, but the particles just drift instead of remaining around where they started. Is there any way to achieve this?

**Comment by matteobruni** [comment]
Why enable wobble? What should the initial particles do? Can you describe more what are you trying to achieve?

**Comment by ldorigo** [comment]
I want to define a list of manual particles that don't move around the page freely but stay close to their initial position (I want to make a recognizable shape with those particles). It's possible to do this by setting `move: { enabled: false} `, but it would look nicer if they did move a bit (wobble) - but remaining centered around their initial location. Let me know if it's not clear, I'll try to explain better or make a drawing :-)

**Reply by matteobruni** [reply]
https://codepen.io/matteobruni/pen/VwgBqRY

Something like this?
